apps/gp2-server/src/data-providers/working-group.data-provider.ts

```diff
@@ -32,15 +56,53 @@ export class WorkingGroupSquidexDataProvider
     return {
       total: result.queryWorkingGroupsContentsWithTotal.total,
       items: result.queryWorkingGroupsContentsWithTotal.items.map(
-        (workingGroup) => ({
-          ...workingGroup.flatData,
-          id: workingGroup.id,
-          title: workingGroup.flatData.title || '',
-          shortDescription: workingGroup.flatData.shortDescription || '',
-          leadingMembers: workingGroup.flatData.leadingMembers || '',
-          members: [],
-        }),
+        parseWorkingGroupToDataObject,
       ),
     };
   }
 }
+type Member = NonNullable<GraphQLWorkingGroup['flatData']['members']>[number];
+
+const parseWorkingGroupMembers = (
+  membersList: gp2.WorkingGroupMember[],
+  member: Member,
+) => {
+  if (!isWorkingGroupMemberRole(member.role)) {
+    throw new Error(`Invalid working group role on members : ${member.role}`);
+  }
+  const user = member.user && member.user[0];
+  if (!user) {
+    return membersList;
+  }
+
+  const flatAvatar = user.flatData.avatar || [];
```

Comment:
once you do what I wrote above you can use that mapper from this point (this is where filtering stops and mapping starts).

Reply:
done